Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN \
 
 # Copy the wrapper script into the container
 COPY bootstrap-script.sh /usr/local/bin/bootstrap-script.sh
+COPY hybrid-ssm-setup.sh /usr/local/bin/hybrid-ssm-setup.sh
```

Comment:
Names should be prefixed with `eks-` to indicate the intended use case. Also, it'd be good to update `README.md` with a description of each helper command and an example of usage.